go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/google/dranet
 
 go 1.24.0
 
+replace k8s.io/dynamic-resource-allocation => github.com/everpeace/kubernetes/staging/src/k8s.io/dynamic-resource-allocation v0.0.0-20250722015156-872f0682d585
```

Comment:
??
didn't the kubernetes upstream already merged?

we can't use this from a private fork

Reply:
Fixed but now running into https://github.com/kubernetes/apimachinery/issues/190

My newest commit fixes this by pinning all the k8s libraries to `v0.34.0-beta.0` that need it but I think it may be better to just wait until the full 1.34 release of K8s for this. Wdyt @aojea?

Reply:
yeah, that sounds the best option, having an intermediate state does not have much sense, also we'll need to update everything to the GA API and modify the examples, since there were changes from v1beta1 to v1beta2 that does not work, per example, request now have `FirstAvailable` and `Exactly`